### PR TITLE
NEWS: Release notes for 0.12

### PR DESCRIPTION
**Pull request by tgraf**
Signed-off-by: Thomas Graf <thomas@cilium.io>

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -138,7 +205,7 @@ Features
 * Reduced use of cgo in in bpf packages (1275_)
 * Do sparse checks during BPF compilation (1175_)
 * New `cilium bpf lb list` command (1317_)
-* New optimized kvstore interaction code (1365_, 1397_, 1370_)
+* New optimized kvstore interaction code (1365_, 1397_, 1370_)1474_)
```
Comment (aanm) [inline NEWS.rst]: `s/)1474_)/1474_)/`

File: .AUTHORS.aux [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (aanm) [inline .AUTHORS.aux:1]: Is this file a leftover?

File: AUTHORS [hunk 3]
```diff
@@ -1,43 +1,55 @@
...
+Jan-Erik Rediger <janerik@fnordig.de>
+Jarno Rajahalme <jarno@covalent.io>
+Joe Stringer <joe@covalent.io>
+John Fastabend <john.fastabend@gmail.com>
+Jonathan Davies <jpds@protonmail.com>
+Jon Davies <jpds@protonmail.com>
+jrajahalme <jarno@covalent.io>
+Katarzyna Borkmann <kasia@iogearbox.net>
+Maciej Kwiek <maciej.iai@gmail.com>
+Madhu Challa <madhu@cilium.io>
```
Comment (aanm) [inline AUTHORS:33]: Madhu is repeated

File: AUTHORS [hunk 4]
```diff
@@ -1,43 +1,55 @@
...
+Eloy Coto <eloy.coto@gmail.com>
+f0 <f0@users.noreply.github.com>
+Faiyaz Ahmed <faiyaza@gmail.com>
+Fred Hsu <fredlhsu@gmail.com>
+genbit <sergey@genbit.ru>
+Gianluca Arbezzano <gianarb92@gmail.com>
+Ian Vernon <ian@covalent.io>
+Ivar Lazzaro <ivarlazzaro@gmail.com>
+Jan-Erik Rediger <janerik@fnordig.de>
+Jarno Rajahalme <jarno@covalent.io>
```
Comment (aanm) [inline AUTHORS:25]: Jarno is repetead

File: AUTHORS [hunk 5]
```diff
@@ -1,43 +1,55 @@
...
-Ross Guarino          rssguar@gmail.com
-Russell Bryant        russell@russellbryant.net
-Sergey Generalov      sergey@genbit.ru
-Steven Ceuppens       steven.ceuppens@icloud.com
-Thomas Bachman        tbachman@yahoo.com
-Thomas Graf           thomas@cilium.io
-Tobias Klauser        tklauser@distanz.ch
-Tony Lambiris         tony@criticalstack.com
-Trevor Roberts Jr     trevor.roberts.jr@gmail.com
+4ast <alexei.starovoitov@gmail.com>
```
Comment (aanm) [inline AUTHORS:4]: `s/4ast/Alexei Starovoitov/`?

File: AUTHORS [hunk 6]
```diff
@@ -1,43 +1,55 @@
...
+Cynthia Thomas <techcet@users.noreply.github.com>
+Daniel Borkmann <daniel@iogearbox.net>
+Daniel Dao <dqminh89@gmail.com>
+Dan Wendlandt <dan@covalent.io>
+Dom Del Nano <ddelnano@gmail.com>
+Eloy Coto <eloy.coto@gmail.com>
+f0 <f0@users.noreply.github.com>
+Faiyaz Ahmed <faiyaza@gmail.com>
+Fred Hsu <fredlhsu@gmail.com>
+genbit <sergey@genbit.ru>
```
Comment (aanm) [inline AUTHORS:20]: `s/genbit/Sergey`?

File: AUTHORS (hunk 5 above)
Comment (tgraf) [reply AUTHORS:4]: This is automatically generated now based on git commits.

File: .AUTHORS.aux (hunk 2 above)
Comment (tgraf) [reply .AUTHORS.aux:1]: This is the static part of the AUTHORS file that is not generated from the git log.

File: NEWS.rst (hunk 1 above)
Comment (tgraf) [reply NEWS.rst]: fixed

**Review by aalemayhu — COMMENTED** [review]
Agree with André, could be cleaned otherwise LGTM.

**Pull request merged**